models/user.go

```diff
@@ -51,7 +52,11 @@ func (hashList HashList) ToStringSlice() []string {
 	return hashList // []string(hashList)
 }
 
-const userTokenSize = 16
+const (
+	userTokenSize = 16
+	//one year in seconds
+	oneYear = 60 * 60 * 24 * 365
```

Comment:
One year seems excessively long. Why dont we keep it small like ~ 1 hour. There is no need for a user to be logged in for any longer.
```suggestion
	oneHour = 60 * 60
```

Reply:
Set expiration to one hour from creation.